pkgs/applications/office/paperless-ng/default.nix

```diff
@@ -5,7 +5,6 @@
 , ghostscript
 , imagemagick
 , jbig2enc
-, ocrmypdf
```

Comment:
Does paperless-ng still work without ocrmypdf? I haven't used it in a while (and it seems to fail to build anyway), but seems worth checking with users if there are any left.

Reply:
As far as I know this is still needed, yes (see their [requirements](https://github.com/jonaswinkler/paperless-ng/blob/7bc8325df910ab57ed07849a3ce49a3011ba55b6/requirements.txt#L58)).

Reply:
also regarding

> seems worth checking with users if there are any left

why wouldn't there be any users left? Am I missing something? :sweat_smile:

Reply:
agh, sorry, this came out more snide than I intended: I'd been using paperless-ng a while ago, and got a bit frustrated with how often its build broke, so I switched away to docspell. It's very possible there are people using it!

Reply:
I've just built it successfully from nixpkgs's master branch, no problems here :/

Reply:
Paperless-ng will still use `ocrmypdf`, just from `python3Packages`.